client/package.json

```diff
@@ -13,7 +13,8 @@
   },
   "dependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "grommet-icons": "^4.12.1"
```

Comment:
when i run front end from scratch after deleting "node_modules" i get this error:
```
[client]     ../node_modules/grommet-icons/es6/utils.js:3:29:
[client]       3 │ import { ThemeContext } from 'styled-components';
```
it says, that transitive dependency "styled-components" cannot be found.
this needs to be added into client's package json manually.